### ISSUE TITLE
mod installation problem

### ISSUE THREAD

**Issue by rodas007** [bgauduch/call-of-duty-2-docker-server#54]
Hi @bgauduch , I'm having trouble installing the ZPAM3 mod, https://github.com/eyza-cod2/zpam3
The mod has to have the following structure for it to work:
![ftp](https://user-images.githubusercontent.com/6936258/80987663-ab335b00-8e32-11ea-9859-3272c75f7c3b.jpg)

I modified the file docker-compose.yaml, adding in : 
`volumes: - ./cod2server/main:/home/cod2/mods/zpam310_beta2`
and command:
`"+set fs_game",
 "mods/zpam310_beta2",`

The server works correctly, but the mod gives errors in the log, attached log to see if we can solve the problem, this same mod I have it working correctly with the following structure in a normal linux server
attached reports

[LOG SERVER.txt](https://github.com/bgauduch/call-of-duty-2-docker-server/files/4575837/LOG.SERVER.txt)

Edit:
I have spoken with the plugin developer and he tells me the following:

``Hi, what hosting do you use? There is a problem that iwd files are loading twice, this is caused by fs_homepath command``

**Comment by bgauduch** [comment]
Hello @rodas007 !

You docker-compose modifications looks good to me !
I've read your logs and it seems that the `sv_referencedIwdNames` variable is to long, but as it is directly related to the server binary I'm not sure what to do to fix this 🤔 

Good idea to ping the plugin developer, an issue regarding `fs_basepath` and `fs_homepath` is already open #33.

Maybe making a proper separation between this two paths will do the trick, not sure though !

**Comment by rodas007** [comment]
perfect, whatever I discover I share with you, to see if we find a solution for this.

**Comment by bgauduch** [comment]
@rodas007 did you manage to make the mod work ?

**Comment by rodas007** [comment]
> @rodas007 did you manage to make the mod work ?

no, it doesn't work for me, I have to try again, I don't know if with the last update you made it already works, I have to try again

**Comment by bgauduch** [comment]
Okay @rodas007, will also try to install the zpam mod on my side when I do have time to see if I can make it work.

I believe the non-root user suppression won't affect this issue thought, I think it's mostly related to the `fs_homepath` setup.

**Comment by rodas007** [comment]
> 
> 
> Okay @rodas007, will also try to install the zpam mod on my side when I do have time to see if I can make it work.
> 
> I believe the non-root user suppression won't affect this issue thought, I think it's mostly related to the `fs_homepath` setup.

**Comment by rodas007** [comment]
> 
> 
> Okay @rodas007, will also try to install the zpam mod on my side when I do have time to see if I can make it work.
> 
> I believe the non-root user suppression won't affect this issue thought, I think it's mostly related to the `fs_homepath` setup.

perfect! As soon as I can try it I tell you, but surely the problem is the fs_homepath, thanks for continuing to work on solving it :)

EDIT: Sorry and inadvertently closed the thread :(

**Comment by bgauduch** [comment]
@rodas007 don't worry, miss-click also happen to me all the time 😄

**Comment by gradisarjoze** [comment]
Have you guys had any luck configuring the pam mod?

**Comment by rodas007** [comment]
> 
> 
> Have you guys had any luck configuring the pam mod?

The zpam3 no longer does not fit, the classic zpam, the previous version does work perfectly.

**Comment by gradisarjoze** [comment]
> > 
> 
> > 
> 
> > Have you guys had any luck configuring the pam mod?
> 
> 
> 
> The zpam3 no longer does not fit, the classic zpam, the previous version does work perfectly.
> 
> 
> 
> 

Are you having any issues with the punkbuster? We are experiencing DUPLICATE GUID/CDKEY frequently and the sv_releaseguid is set to 7.

**Comment by rodas007** [comment]
> 
> 
> > > 
> > 
> > 
> > > 
> > 
> > 
> > > Have you guys had any luck configuring the pam mod?
> > 
> > 
> > The zpam3 no longer does not fit, the classic zpam, the previous version does work perfectly.
> 
> Are you having any issues with the punkbuster? We are experiencing DUPLICATE GUID/CDKEY frequently and the sv_releaseguid is set to 7.

no, what we have been able to test on several servers worked correctly.

**Comment by bgauduch** [comment]
@rodas007 and @gradisarjoze , closing this issue as it is inactive.

Feel free to reopen if needed !
